content/documentation/3-developer-guides/3-batch/1-simple-task.md

```diff
@@ -11,31 +11,293 @@ In this guide we will develop a batch application using Spring Cloud Task and de
+1. BillRun will be a Spring Boot application using Spring Cloud Task and Spring Batch that will read data from `BILL_USAGE` table and price the each row and put the resulting data into the `BILL_STATEMENTS` table.
+
+'For this section we will create a Spring Cloud Task/Boot application that will read JSON files containing customer usage data from a S3 bucket and write the raw data to a table in the database.  
+The data contained in the files will be: id, customer first name , customer last name, cellular minutes, and data usage. The format of the data looks like this
+
+```
+[{"id":"16","firstName":"jane","lastName":"doe","minutes":"500","dataUsage":"1000"},
+{"id":"20","firstName":"mary","lastName":"jones","minutes":"650","dataUsage":"1500"}]
+```
 
-Create a simple task (timestamp)
+This usage data will then be inserted into a `BILL_USAGE` table that looks like the following:
+
+```
+CREATE TABLE BILL_USAGE
+(
+   id int,
+   first_name varchar(50),
+   last_name varchar(50),
+   minutes int,
+   data_usage int
+)
+```
 
 ### Initialzr
 
-Go to initializr - maybe have a curl command ready to go do we can just download a ready made .zip.
+1. Visit the Spring Initialzr site.
+1. Select the latest release of spring boot
+1. Create a new Maven or Gradle project with a Group name of io.spring and an Artifact name of billusage.
+1. In the Dependencies text box, type task to select the Cloud Task dependency.
+1. In the Dependencies text box, type jdbc then select the JDBC dependency.
+1. In the Dependencies text box, type h2 then select the H2.
+1. In the Dependencies text box, type mysql then select mysql (or your favorite database)
+1. Click the Generate Project button
+1. Unzip the billusage.zip file and import the project into your favorite IDE.
+1. From your favorite IDE add the following dependeny to your pom.xml
+   ```
+    <dependency>
+        <groupId>com.fasterxml.jackson.core</groupId>
+        <artifactId>jackson-databind</artifactId>
+    </dependency>
+   ```
 
 ### Biz Logic
 
-Your biz logic is coded in this section,
+Now let’s create the elements required for this application.
+
+1.  Using your IDE create a usagedata.json file in the resources directory.
+    1. Copy and paste the contents below into the usagedata.json file:
+       ```
+       [{"id":"1","firstName":"jane","lastName":"doe","minutes":"500","dataUsage":"1000"},
+         {"id":"2","firstName":"john","lastName":"doe","minutes":"550","dataUsage":"1500"},
+         {"id":"3","firstName":"melissa","lastName":"smith","minutes":"600","dataUsage":"1550"},
+         {"id":"4","firstName":"michael","lastName":"smith","minutes":"650","dataUsage":"1500"},
+         {"id":"5","firstName":"mary","lastName":"jones","minutes":"650","dataUsage":"1500"}]
+       ```
+1.  In your favorite IDE create a `configuration` package under the `io.spring.billusage` package
```

Comment:
Packages in java are actually not hierarchal see ([Apparent Hierarchies of Packages,](https://docs.oracle.com/javase/tutorial/java/package/usepkgs.html)) they are just flat strings that just happen to look like they are hierarchal so we should just say "create a package `io.spring.billusage.configuration`".

Reply:
updated